src/main/java/org/jenkinsci/plugins/mesos/api/LaunchCommandBuilder.java

```diff
@@ -174,4 +183,12 @@ private URL buildJnlpUrl() throws MalformedURLException {
         .add(jenkinsAgentFetchUri)
         .build();
   }
+
+  private static DomainFilter ANY_DOMAIN =
```

Comment:
I would suggest having this available from USI rather than defining it here, that way any framework can pick between `AnyDomainFilter`, `HomeRegionFilter`, `RegionFilter` etc... and we build up a library of those ready for use, in a "batteries included" sort of manner.

Reply:
Good point. There is a follow up at https://github.com/mesosphere/usi/pull/127.